packages/commonwealth/shared/adapters/chain/ethereum/types.ts

```diff
@@ -34,6 +34,15 @@ export class ERC721Token extends EthereumCoin {
   }
 }
 
+export class ERC1155Token extends EthereumCoin {
```

Comment:
This doesnt appear to be use - do we need this for something?

Reply:
Removed this, Don't think we need it